### ISSUE TITLE
False motion/open status

### ISSUE THREAD

**Issue by abmantis** [louisZL/lumi-gateway-local-api#11]
Using this api, many times the gateway reports sensors as "open" or "motion", when there is no motion or door open. The sensor will stay in the "open" or "motion" state indefinitely until manually triggered (door opens or real motion happens).

On Mi Home app status is always correct (no false status).

**Comment by ravetam** [comment]
This is an API issue. Gateway is broadcasting the wrong data. There's no issue with the component itself. Nothing can be done here unless xiaomi fix the API in the firmware. You can follow up on this issue on HomeAssistant forum

**Comment by ravetam** [comment]
Just noticed this is not my repo...

**Comment by dimalo** [comment]
please check if its a heartbeat message and not a report message. with heartbeats you have to ignore the status...

**Comment by abmantis** [comment]
@dimalo Yes, it is only on heartbeat messages. Yeah, I can ignore them, but it is a bug anyway.
